dapp_runner/__main__.py

```diff
@@ -20,7 +20,7 @@
 logger = logging.getLogger(__name__)
 
 
-@click.group
+@click.group()
```

Comment:
?

Reply:
IDK, It doesn't work without `()` for me for some reason.

Reply:
ah